local-cli/generator/index.js

```diff
@@ -8,11 +8,40 @@
  */
 'use strict';
 
+var execSync = require('child_process').execSync;
 var fs = require('fs');
 var path = require('path');
+var semver = require('semver')
 var utils = require('../generator-utils');
 var yeoman = require('yeoman-generator');
 
+// Use Yarn if available, it's much faster than the npm client.
+// Return the version of yarn installed on the system, null if yarn is not available.
+function getYarnVersionIfAvailable() {
+  let yarnVersion;
+  try {
+    // execSync returns a Buffer -> convert to string
+    if (process.platform.startsWith('win')) {
```

Comment:
Good to know, thanks! Wanted to be safe in case some future node versions start returning 'win64'. Even though they shouldn't as that would be a breaking change.